yt/frontends/ramses/data_structures.py

```diff
@@ -254,13 +353,14 @@ def _detect_output_fields(self):
         self.field_list = self.particle_field_list + self.fluid_field_list
 
     def _identify_base_chunk(self, dobj):
+        ngz = dobj._num_ghost_zones
```

Comment:
```suggestion
```

Reply:
I'm addressing this one on #2425 where it first appeared.